geode/src/test/java/org/apache/calcite/adapter/geode/rel/GeodeBookstoreTest.java

```diff
@@ -455,13 +459,16 @@ public void testSqlLimit2() throws SQLException {
 
 
   @Test
-  public void testSqlDisjunciton() throws SQLException {
+  public void testSqlDisjunction() throws SQLException {
+    String expectedQuery = "SELECT author AS author FROM /BookMaster "
+        + "WHERE itemNumber IN SET(123, 789)";
```

Comment:
Can you add test for booleans and floats as well ? 

Also add tests for longer lists (eg. all 50 states).

Tests for combination of `INs` in the same query:
```sql
foo in (1, 2, 3) and bar in (true, false) or qux in ('q1', 'q2')
```

Reply:
Can I add a boolean field to book_master.json or book_customer.json to test boolean fields.?
Right now there are no boolean fields in there

Reply:
Can you check if `book_master.json` or `book_customer.json` somewhere else ? They were taken from `calcite-test-dataset`.

Probably it will be simpler to create new region with all types (float, String, boolean etc.)